Tasks/NuGetInstaller/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,27 +1,23 @@
 {
-  "loc.friendlyName": "NuGet Installer",
+  "loc.friendlyName": "NuGet Restore",
   "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=613747)",
-  "loc.description": "Restores missing NuGet packages",
+  "loc.description": "Restores NuGet packages in preparation for a Visual Studio Build step.",
```

Comment:
It's uppercased because it's talking about the "Visual Studio Build" step I think.